templates/registration/login-head.html

```diff
@@ -0,0 +1,34 @@
+{% load static %}
+
+<!-- begin::Head -->
+
+<head>
```

Comment:
can we use the same head we are using in base ?

Reply:
@Ayub-Khan sir both heads have some different references as you can see [here](https://www.diffchecker.com/IvVlH5Kk).